drydock_provisioner/objects/__init__.py

```diff
@@ -28,6 +28,7 @@ def register_all():
     __import__('drydock_provisioner.objects.hostprofile')
     __import__('drydock_provisioner.objects.hwprofile')
     __import__('drydock_provisioner.objects.site')
+    __import__('drydock_provisioner.objects.promenade')
```

Comment:
You could consider using the more modern `importlib.import_module()` instead of `__import__`.